client/src/pages/Profile/ProfileListings/ProfileListings.tsx

```diff
@@ -0,0 +1,137 @@
+        "name": "Oneida Latek",
+        "catch_phrase": "Re-contextualized web-enabled moratorium",
+        "description": "Nullam porttitor lacus at turpis. Donec posuere metus."
+      }, 
+      {
+        "image_url": "https://robohash.org/solutaametratione.png?size=50x50&set=set1",
+        "name": "Kare Lemonby",
+        "catch_phrase": "Open-source grid-enabled benchmark",
+        "description": "Nulla ut erat id mauris vulputate elementum. Nullam."
+      }
+    ]
+
+    return(
+        <Grid container className={classes.root}>
+            <Navbar
+                LOGOUT="/logout"
+                PROFILE="/profile"
+                MY_SITTER="/mySitter"
+                BECOME_SITTER="/becomeSitter"
+                MESSAGE="/message"
+                Logout="Log out"
+                Profile="Profile"
+                Mysitter="My Sitters"
+                Becomesitter="BECOME A SITTER"
+                Messages="Messages"
+            />
+            <Grid item xs={12}>
+                <Grid container direction="column">
+                    <Grid item>
+                        <Box justifyContent="center" >
+                            <Typography variant="h4" align="center">Your search results</Typography>
+                            <Box display="flex" justifyContent="center" mt={3}>
+                                <TextField 
+                                    label="Toronto, Ontario" 
+                                    margin="normal" 
+                                    variant="outlined" 
+                                    InputProps={{
+                                        startAdornment: (
+                                            <InputAdornment position="start">
+                                            <Search />
+                                            </InputAdornment>
+                                        ),
+                                    }}
+                                />
+                                <TextField 
+                                    margin="normal" 
+                                    variant="outlined" 
+                                    type="date"
+                                />
+                            </Box>
+
+                            <Box display="flex" flexWrap="wrap" justifyContent="center" pt={5} className={classes.profileArea}>
+                                {
+                                    data.map((d, index) => {
+                                        return(
+                                            <Link key={index} to="/profile-details" style={{textDecoration: 'none'}}>
+                                                <Card className={classes.cardWrapper}>
+                                                    <CardActionArea>
+                                                        <Avatar className={classes.cardAvatar} alt="Remy Sharp" src={d.image_url} />
```

Comment:
The alt tags could be replaced with something from the data, maybe to something like `` {`Avatar of ${d.name}`} ``?
Also, let's rename `image_url` to be in `camelCase`.